files/salt/openSUSE-15.0-x86_64_tiny/root/.bash_history

```diff
@@ -0,0 +1,36 @@
+salt-run state.orch ceph.stage.0
```

Comment:
If you didn't change something, this file should end up on the node anyways. You don't need to specify this explicitly.